back/app/services/survey_results_generator_service.rb

```diff
@@ -165,8 +169,8 @@ def visit_select_base(field)
     query = inputs
     query = query.joins(:author) if group_mode == 'user_field'
     if group_field
-      raise "Unsupported group field type: #{group_field.input_type}" unless %w[select linear_scale rating].include?(group_field.input_type)
-      raise "Unsupported question type: #{field.input_type}" unless %w[select multiselect linear_scale rating multiselect_image].include?(field.input_type)
+      raise "Unsupported group field type: #{group_field.input_type}" unless %w[select linear_scale sentiment_linear_scale rating].include?(group_field.input_type)
+      raise "Unsupported question type: #{field.input_type}" unless %w[select multiselect linear_scale sentiment_linear_scale rating multiselect_image].include?(field.input_type)
```

Comment:
Got a lot of these scattered through the survey generator, but let's not tackle here. I have some refactoring of this service that I want to do before we add the date dimensions so I'll do this then